CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -952,6 +952,7 @@ local function CreateSettingsHub()
 		local panel = Panel3D.Get(Panel3D.Panels.Settings)
```

Comment:
this in general feels confusing to have a thing named panel3d and then panel, when it is the settings. Also, would it make sense to have all these panel configs (modal, orientation, cursorenabled) in a function? I don't know exactly what we are doing here but it seems like the 3d settings panel should be able to do this stuff internally